.travis.yml

```diff
@@ -106,14 +106,17 @@ before_install:
           local ext_name=$1
           local ext_so=$2
           local INI=$3
+          local ext_opt=$4
           local ext_dir=$(php -r "echo ini_get('extension_dir');")
           local ext_cache=~/php-ext/$(basename $ext_dir)/$ext_name
 
           if [[ -e $ext_cache/$ext_so ]]; then
               echo extension = $ext_cache/$ext_so >> $INI
           else
               mkdir -p $ext_cache
-              echo yes | pecl install -f $ext_name &&
+              pecl config-show
+              pecl config-set libzookeeper-dir /zookeeper-3.4.12
```

Comment:
Thanks for suggestion. I was looking for a way to provide the option for pecl itself when it runs configure. Is there no way to do that?

Reply:
Sorry, I'm not familiar with command `pecl`.
But IMO, I don't think it can provide actions you want.

I read `.travis.yml`, and I found that you may install `zookeeper-dev` (Or other name with which the package contains header files of `libzookeeper`) via `apt` just like what pecl `libsodium` did.
:)